test/any.js

```diff
@@ -21,7 +21,7 @@ describe('any', function() {
     eq(intoArray(R.any(odd), [2, 4, 6, 8, 10, 11, 12]), [true]);
   });
 
-  it('returns false if all elements fails to satisfy the predicate', function() {
+  it('returns false into array if all elements fails to satisfy the predicate', function() {
```

Comment:
@CrossEye: I'm not sure we even need those two last tests. Unless that was testing the transducer nature of `R.any` somehow but I don't see how. It seems like this is testing `R.into` more than it is testing `R.any`.

Reply:
Agreed.  These test seem superfluous.

Reply:
Just to clear my understanding,
If that is true, you can see there are a lot of places testing like this.
Such as inside of all / find / map, etc.

Maybe we should move all of such tests into `into`?

Reply:
Thanks!
I removed all into related tests,
and add at least one single test for each functions(any/all/find/findIndex/findLast/findLastIndex/chain/map/none).

The only one I kept is [this in chain](https://github.com/ramda/ramda/blob/master/test/chain.js#L59-L64),
I don't know whether I should remove it or not.

Kindly have a check.